### PR TITLE
Update signature attribute in claim details API.

### PR DESCRIPTION
**Pull request by prashantramangupta**

### CODE REVIEW

**Review by anandrgitnirman — APPROVED** [review]

File: airdrop/application/handlers/airdrop_handlers.py [hunk 1]
```diff
@@ -64,10 +64,9 @@ def user_eligibility(event, context):
 
 
 @exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger)
-def airdrop_window_claims(event, context):
-    logger.info(f"Got Airdrops Window Claims Events {event}")
-    status, response = AirdropServices(
-    ).airdrop_window_claims(request(event))
+def occam_airdrop_window_claim(event, context):
```
Comment (raamb) [inline airdrop/application/handlers/airdrop_handlers.py:67]: Ideally this should have been part of the dynamic class

File: airdrop/constants.py [hunk 2]
```diff
@@ -24,10 +24,22 @@
     },
     "required": ["signature", "address", "airdrop_id", "airdrop_window_id", "block_number"],
 }
+OCCAM_CLAIM_SCHEMA = {
```
Comment (raamb) [inline airdrop/constants.py:27]: OCCAM_CLAIM_SCHEMA and CLAIM_SCHEMA are the same? Do we need different structures?

File: airdrop/application/handlers/airdrop_handlers.py (hunk 1 above)
Comment (prashantramangupta) [reply airdrop/application/handlers/airdrop_handlers.py:67]: Yes you are right. Occam API's are different and it is clearly mentioned in comments that delete the code once Occam is done.
So I didn't created dynamic class for Occam.

File: airdrop/application/services/airdrop_services.py [hunk 3]
```diff
@@ -359,36 +371,36 @@ def airdrop_window_claim(self, inputs):
...
+            response = {
+                "airdrop_id": airdrop_id,
+                "airdrop_window_id": airdrop_window_id,
+                "user_address": user_address,
+                "signature": signature,
+                "claimable_amount": str(claimable_amount),
+                "token_address": token_address,
+                "contract_address": contract_address,
+                "staking_contract_address": staking_contract_address,
+                "total_eligibility_amount": str(total_eligibility_amount)
```
[the remark this answers is not in the document]
Comment (prashantramangupta) [reply airdrop/application/services/airdrop_services.py:402]: The Python "TypeError: Object of type Decimal is not JSON serializable"
I can think of two options
1. Extend the JSONEncoder class and convert the decimal to a string to preserve its precision.
2. Or convert the decimal value to string.
The existing flow make use of 2nd option.

File: airdrop/constants.py (hunk 2 above)
Comment (prashantramangupta) [reply airdrop/constants.py:27]: Nope we don't need different structure. Will reuse the one defined structure.

File: airdrop/application/services/airdrop_services.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (raamb) [reply airdrop/application/services/airdrop_services.py:402]: But this was working before. Lets ensure that the frontend does not break

**Pull request merged**